connector/src/main/java/org/geysermc/connector/skin/SkinProvider.java

```diff
@@ -602,6 +667,14 @@ public static BufferedImage imageDataToBufferedImage(byte[] imageData, int image
         private final Cape cape;
     }
 
+    @AllArgsConstructor
+    @Getter
+    public static class SkinData {
```

Comment:
This can be a Java 16 record without the Lombok annotations. Do note that the get prefix is removed when you do this; that's fine.

I just saw this idly; I'll be doing a full review later.